.github/workflows/e2e.yaml

```diff
@@ -30,18 +30,20 @@ jobs:
 
       - name: Create a temporary directory
         id: createpath
+        shell: bash
         run: |
           project_path=$(mktemp -d -t redwood.XXXXXX)
           echo "::set-output name=project_path::$project_path"
 
       - name: Create a RedwoodJS app
+        shell: bash
         run: ./tasks/run-e2e ${{ steps.createpath.outputs.project_path }} --no-start
 
       - name: Start server in background
+        shell: bash
         run: yarn rw dev &
         working-directory: ${{ steps.createpath.outputs.project_path }}
 
```

Comment:
@Tobbe can you help me diagnose _why_ this would fail on Windows bash? Actually, what would be ideal is to instead use scripts for each of these steps that work on Powershell. Possible?

Reply:
I'd try something like this:
```
run: |
  echo "${{ steps.createpath.outputs.project_path }}"
  cd "${{ steps.createpath.outputs.project_path }}"
  yarn rw dev &
```

It'll tell us
1. Do we even get a valid path to this step?
2. Does it look right?
3. Does it help to start the server when standing in the directory as opposed to trying to set cwd

Reply:
@Tobbe well, that worked. Question: what would the Powershell version of this command be (if possible)?

Next step: can I have the Cypress action run with Powershell... gonna go check some docs.

Reply:
`echo` and `cd` are both commands PS understands. So if it's GH that does the `"${{ steps.createpath.outputs.project_path }}"` substitution to a proper value, it should all just work in PS as well.  But if GH is doing some kind of injection of variables into the shell, and it's actually bash that does the substitution then I don't know.